ocaml/idl/datamodel_types.ml

```diff
@@ -334,6 +334,13 @@ let release_order_full =
     ; branding= "Citrix Hypervisor 8.2 Hotfix 2"
     ; release_date= Some "November 2020"
     }
+  ; {
+      code_name= Some "stream"
```

Comment:
Could  we call this "cloud" to match the terms the UI uses internally.

Reply:
Changed to `nile-preview` according to our latest discussion.